Geometry/VeryForwardGeometry/python/dd4hep/v2

```diff
@@ -0,0 +1,18 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
This file does not have a correct name.

Reply:
It should be a dir actually. Something went wrong... I fixed it.